README.md

```diff
@@ -169,31 +113,80 @@ WidgetsBinding.instance.addPostFrameCallback((_) =>
+| enableAutoScroll          | bool                        | false                        | Allows to auto scroll to next showcase so as to make the given target visible. |
+| scrollDuration            | Duration                    | Duration(milliseconds: 300)  | Time duration for auto scrolling                                               |
+| disableBarrierInteraction | bool                        | false                        | Disable barrier interaction                                                    |
+| disableScaleAnimation     | bool                        | false                        | Disable scale transition for all showcases                                     |
+| disableMovingAnimation    | bool                        | false                        | Disable bouncing/moving transition for all showcases                           |
+| onStart                   | Function(int?, GlobalKey)?  |                              | Triggered on start of each showcase.                                           |
+| onComplete                | Function(int?, GlobalKey)?  |                              | Triggered on completion of each showcase.                                      |
+| onFinish                  | VoidCallback?               |                              | Triggered when all the showcases are completed                                 |
+
+## Properties of `Showcase` and `Showcase.withWidget`:
+
+| Name                         | Type          | Default Behaviour                                | Description                                                                                        | `Showcase` | `ShowCaseWidget` |
+|------------------------------|---------------|--------------------------------------------------|----------------------------------------------------------------------------------------------------|------------|------------------|
+| key                          | GlobalKey     |                                                  | Unique Global key for each showcase.                                                               | ✅          | ✅                |
+| child                        | Widget        |                                                  | The Target widget that you want to be showcased                                                    | ✅          | ✅                |
+| title                        | String?       |                                                  | Title of showcase into default tooltip                                                             | ✅          |                  |
+| description                  | String?       |                                                  | Description of showcase into default tooltip                                                       | ✅          |                  |
```

Comment:
Description of showcase into default tooltip -> Description of the default tooltip